.circleci/config.yml

```diff
@@ -15,23 +15,40 @@ jobs:
     steps:
       - checkout
       - terraform/install:
-        terraform_version: 0.13.5
```

Comment:
These changes to `config.yml` will of course need to be incorporated by everybody in @dod-iac/dds. This is a good thing! But I remember that, when I was working on the terratest module, it took me a little while to figure out why my git commits were failing. It was because I had not pulled the latest version of `config.yml` to my branch.

Reply:
I ported these changes back to the `dod-iac/terraform-module-template` repo. Always look there and compare. That should always be your first step in debugging.